src/App.tsx

```diff
@@ -1,60 +1,107 @@
-import classNames from 'classnames';
-
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
 import { UserSelector } from './components/UserSelector';
+import { useEffect, useState } from 'react';
+import { User } from './types/User';
+import { getUsers } from './api/users';
 import { Loader } from './components/Loader';
+import { PostsList } from './components/PostsList/PostsList';
+import { Post } from './types/Post';
+import { getPosts } from './api/posts';
+import { PostDetails } from './components/PostDetails';
```

Comment:
Ensure that the `PostDetails` component is correctly implemented and functional, as it is crucial for displaying the selected post details in the sidebar. Verify that this component exists and is correctly imported.